src/odemis/driver/actuator.py

```diff
@@ -1934,11 +1936,13 @@ def __init__(self, name, role, dependencies, axis_name, cycle=2 * math.pi,
             raise ValueError("RotationActuator needs precisely one dependency")
 
         self._cycle = cycle
-        # counter to check when current position has overrun cycle
-        # and is close to zero again (pos and neg direction)
-        self._move_sum = 0
         # check when a specified number of rotations was performed
+        if ref_on_move_count is None or ref_on_move_count >= 1:
+            self._ref_on_move_count = ref_on_move_count
+        else:
+            raise ValueError("ref_on_move_count is %s, but must be >= 1 or None" % (ref_on_move_count,))
```

Comment:
I don't think it is nice to have variable names in an error message. I suggest to say something like `"Referencing after %s moves requested, but must be ....".`

Reply:
I respectfully disagree! That's the argument name in the microscope file, it's much clearer to know which argument is causing the issue.

Reply:
ok, then it is fine.